### PR TITLE
Strings

### PR DESCRIPTION
**Pull request by luis-pereira**

### CODE REVIEW

File: src/mainwindow.cpp [hunk 1]
```diff
@@ -701,17 +701,17 @@ void MainWindow::loadImage(const Fm::FilePath & filePath, QModelIndex index) {
...
-  if(mimeType == "image/gif"
-     || mimeType == "image/svg+xml" || mimeType == "image/svg+xml-compressed") {
+  if(mimeType == QLatin1String("image/gif")
+     || mimeType == QLatin1String("image/svg+xml") || mimeType == QLatin1String("image/svg+xml-compressed")) {
     const Fm::CStrPtr file_name = currentFile_.toString();
     ui.view->setAutoZoomFit(true); // like in onImageLoaded()
-    if(mimeType == "image/gif")
-      ui.view->setGifAnimation(QString{file_name.get()});
+    if(mimeType == QLatin1String("image/gif"))
+      ui.view->setGifAnimation(QString{QString::fromUtf8(file_name.get())});
```
Comment (tsujan) [inline src/mainwindow.cpp]: There's a redundant `QString`  here and two lines below it.

**Comment by luis-pereira** [comment]
@tsujan pinging before rebase becames necessary.

**Comment by tsujan** [comment]
> @tsujan pinging before rebase becames necessary.

I don't understand. Should I do something?

**Comment by luis-pereira** [comment]
If all is Ok, approve or mege. Or both.

**Comment by tsujan** [comment]
Thanks!

**Comment by luis-pereira** [comment]
Thanks. Rebasing is sometimes a PITA.

**Comment by tsujan** [comment]
> Rebasing is sometimes a PITA.

I agree ;)

**Comment by tsujan** [comment]
And I wouldn't merge any lximage-qt PR before this was merged.

**Pull request merged**